app/src/main/java/org/oppia/android/app/profile/PinPasswordActivityPresenter.kt

```diff
@@ -68,6 +75,16 @@ class PinPasswordActivityPresenter @Inject constructor(
       (activity as PinPasswordActivity).finish()
     }
 
+    outState?.getProto(
+      PINPASSWORD_ACTIVITY_STATE_KEY,
+      PinPasswordActivityStateBundle.getDefaultInstance()
+    )?.inputPin?.let { savedInputPin ->
+      if (savedInputPin.isNotEmpty()) {
+        Toast.makeText(activity, savedInputPin.toString(), Toast.LENGTH_SHORT).show()
```

Comment:
We don't use toasts at all in the project. If the savedstate is null, then the pin textview should just be empty.